gpkit/nomials/nomial_core.py

```diff
@@ -6,24 +6,6 @@
 from ..repr_conventions import _str, _repr, _repr_latex_
 
 
-def fast_monomial_str(exp, c):
```

Comment:
If we're dropping this method, then we should also drop the commented-out code in `gpkit/nomials/nomial_math.py` that calls it.